modules/products/bitbucket/nfs/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "product" {
```

Comment:
How this is used or intended to be used? Is it really necessary? I think if we are using implicit dependency in the `helm_release` resource for the PVC, it should create the nfs module by default?